### PR TITLE
solution

### PR DESCRIPTION
**Pull request by andrii-koptiev**

### CODE REVIEW

File: src/robotFactory.js [hunk 1]
```diff
@@ -1,10 +1,78 @@
...
+}
+
+class FlyingRobot extends BaseRobot {
+  constructor(name, weight, coords, chipVersion) {
+    super(name, weight, coords, chipVersion);
+
+    this.coords = {
+      x: coords.x || 0,
+      y: coords.y || 0,
+      z: coords.z || 0,
```
Comment (marina-tilniak) [inline src/robotFactory.js:45]: x and y already defined in parent, you can just  write this.coords.z = coords.z || 0;

**Review by marina-tilniak — APPROVED** [review]

File: src/robotFactory.js (hunk 1 above)
Comment (andrii-koptiev) [reply src/robotFactory.js]: спасибо. исправил

File: src/robotFactory.js [hunk 2]
```diff
@@ -1,10 +1,74 @@
...
+    this.coords.x += step;
+  };
+
+  goLeft(step = 1) {
+    this.coords.x -= step;
+  };
+
+  getInfo() {
+    // eslint-disable-next-line max-len
+    return `Robot: ${this.name}, Chipversion: ${this.chipVersion} Weight: ${this.weight}`;
```
Comment (Elena-brv) [inline src/robotFactory.js:34]:
```suggestion
    return `Robot: ${this.name},  `
      + `Chipversion: ${this.chipVersion}, ` 
      + `Weight: ${this.weight}`;
```

**Review by Elena-brv — APPROVED** [review]
do not disable linter

**Pull request closed without merging**